translation-definitions/src/main/java/eu/europeana/api/translation/definitions/model/LanguageDetectionObj.java

```diff
@@ -0,0 +1,72 @@
+package eu.europeana.api.translation.definitions.model;
+
+/**
+ * The Data Model class used for holding information used during the processing of Language Detection requests
+ * @author srishti singh
+ * @since 6 Feb 2024
+ */
+public class LanguageDetectionObj implements LanguageObj {
+
+    private String text;
+    private String hint;
+    private String detectedLang;
+    private String cacheKey;
+    private boolean availableInCache;
+    private boolean isTranslated;
```

Comment:
should rename to translated/translatable, but we should clarify with Hugo, which info we want to keep in the object, if alleady processed and decided that the null value is to be return, or if is eligble for translation/detection (having a different meaning for filtering the values sent to the remote services and for building responses)

Reply:
we could have an AbstractLanguageObj to move the commons fields to the base class